patterns/base/li/li.twig

```diff
@@ -0,0 +1,4 @@
+{% for item in li %}
```

Comment:
sorry @rebmullin this repeat doesn't belong here. Only when it's being called by another template. So, the repeat only belongs in the template calling this template.